apps/app/pages/admin/lectures/index.tsx

```diff
@@ -143,14 +153,16 @@ function Lectures() {
           title={lecture.event?.title}
           style={{ maxWidth: 460, margin: 15 }}
           extra={
-            <Button
-              type="primary"
-              shape="circle"
-              icon={<CloseOutlined />}
-              onClick={() => {
+            <Popconfirm
+              title="Tem certeza que deseja excluir essa sessão?"
```

Comment:
Portuguese can be hard sometimes :)
```suggestion
              title="Tens a certeza que queres eliminar esta sessão?"
```